hypervisor/include/arch/x86/guest/vm.h

```diff
@@ -63,6 +63,7 @@ struct vm_pm_info {
 	struct cpu_px_data	px_data[MAX_PSTATE];
 	uint8_t			cx_cnt;		/* count of all Cx entries */
 	struct cpu_cx_data	cx_data[MAX_CSTATE];
+	struct pm_s_state_data	*s_state_data;
```

Comment:
rename s_state_data to sx_state_data; and add a comment to mention this is for S3/S5 xxxx ....